CHANGELOG.md

```diff
@@ -12,39 +12,79 @@
 
 ### Minor Changes
 
-- 2a75b9f: Removes `GuWazuhAccess` security group as Wazuh has been deprecated.
-
-  This change will remove a resource of logical ID `WazuhSecurityGroup` from stacks that use a `GuAutoScalingGroup`.
-  The snapshot diff will include the removal of the following resource:
-
-  ```json
-  {
-    "Resources": {
-      "WazuhSecurityGroup": {
-        "Properties": {
-          "GroupDescription": "Allow outbound traffic from wazuh agent to manager",
-          "SecurityGroupEgress": [
-            {
-              "CidrIp": "0.0.0.0/0",
-              "Description": "Wazuh event logging",
-              "FromPort": 1514,
-              "IpProtocol": "tcp",
-              "ToPort": 1514
-            },
-            {
-              "CidrIp": "0.0.0.0/0",
-              "Description": "Wazuh agent registration",
-              "FromPort": 1515,
-              "IpProtocol": "tcp",
-              "ToPort": 1515
-            }
-          ],
-          "Type": "AWS::EC2::SecurityGroup"
-        }
+Removes `GuWazuhAccess` security group as Wazuh has been deprecated (#2561).
+
+This change will remove a resource of logical ID `WazuhSecurityGroup` from stacks that use a `GuAutoScalingGroup`.
+The snapshot diff will include the removal of the following resource:
+
+```json
+{
+  "Resources": {
+    "WazuhSecurityGroup": {
+      "Properties": {
+        "GroupDescription": "Allow outbound traffic from wazuh agent to manager",
+        "SecurityGroupEgress": [
+          {
+            "CidrIp": "0.0.0.0/0",
+            "Description": "Wazuh event logging",
+            "FromPort": 1514,
+            "IpProtocol": "tcp",
+            "ToPort": 1514
+          },
+          {
+            "CidrIp": "0.0.0.0/0",
+            "Description": "Wazuh agent registration",
+            "FromPort": 1515,
+            "IpProtocol": "tcp",
+            "ToPort": 1515
+          }
+        ],
+        "Type": "AWS::EC2::SecurityGroup"
       }
     }
   }
-  ```
+}
+```
+
+#### How to update to this version
+This version of `@guardian/cdk` detaches the `WazuhSecurityGroup` security group from any autoscaling group and deletes it in one step.
+When using [Riff-Raff's autoscaling deployment type](https://riffraff.gutools.co.uk/docs/magenta-lib/types#autoscaling), upgrading needs to be performed in two steps.
+If we do not, Riff-Raff will fail with an error similar to:
+
+> WazuhSecurityGroup(AWS::EC2::SecurityGroup}: DELETE_FAILED resource sg-1234 has a dependent object (Service: Ec2, Status Code: 400) (SDK Attempt Count: 1)
+
+1. Detach the `WazuhSecurityGroup` security group from the autoscaling group
+
+   In this step, we detach the `WazuhSecurityGroup` security group from the autoscaling group by upgrading to v61.5.0
```

Comment:
Can you help me understand what you mean by detaching the security group from the ASG? Like you would detach an instance from an ASG? Or do you mean removing it from the lists of SGs in the launch template?